docs/querying/lookups.md

```diff
@@ -347,7 +347,7 @@ These end points can be used to get the propagation status of configured lookups
 
 ### List lookup state of all processes
 
-`GET /druid/coordinator/v1/lookups/nodeStatus` with optional query parameter `discover` to discover tiers or configured lookup tiers are listed.
+`GET /druid/coordinator/v1/lookups/nodeStatus` with optional query parameter `discover` to discover tiers or configured lookup tiers are listed. The default response will include the which lookups are loaded, being loaded, or being dropped on each node for each tier, including the complete lookup spec. Add the optional query parameter `detailed=false` to only include the 'version' of the lookup instead of the complete spec.
```

Comment:
nit:
```suggestion
`GET /druid/coordinator/v1/lookups/nodeStatus` with optional query parameter `discover` to discover tiers or configured lookup tiers are listed. The response will include the lookups which are loaded, being loaded, or being dropped on each node for each tier. By default, complete lookup spec is included in response for each lookup. Add the optional query parameter `detailed=false` to only include the 'version' of the lookup instead of the complete spec.
```